src/products/review/signoffs.md

```diff
@@ -2,24 +2,40 @@
 
 <!-- toc -->
 
-Sign-offs are a way for l10n-drivers to indicate that a specific changeset is technically sound and ready to ship in Firefox desktop or Firefox for Android. Currently, thanks to cross-channel, we ship all versions of Firefox from a [single localization repository](https://hg.mozilla.org/l10n-central/), but sign-offs only happen for the Beta channel:
+Sign-offs are a way for l10n-drivers to indicate that a specific changeset is technically sound and ready to ship in Firefox desktop or Firefox for Android. Currently, thanks to cross-channel, we ship all versions of Firefox from a [single localization repository](https://hg.mozilla.org/l10n-central/), but sign-offs are only relevant for the Beta channel:
```

Comment:
I'd drop the last half-sentence. It's less accurate than the previous version, and also doesn't match the sign-offs on Nightly during RC week. And with the 4-week cycle, it's a significant amount of time.